src/frontend/screens/AllProjects.tsx

```diff
@@ -43,63 +75,58 @@ export const AllProjects: NativeNavigationComponent<'AllProjects'> = () => {
         }}
         renderIcon={() => <AddProjectIcon />}
       />
-      <FlatList
-        style={{padding: 20}}
+      <FlatList<ProjectListItem>
+        contentContainerStyle={{padding: 20}}
         data={data}
-        renderItem={({item}) => {
-          function handlePress() {
-            if (currentProjectId === item.projectId) {
-              popTo('Menu');
-              return;
-            }
-
-            if (isTracking && currentProjectId !== item.projectId) {
-              navigate('TrackRecordingActive');
-              return;
-            }
-            setActiveProjectId(item.projectId);
-            popTo('Menu');
-          }
-          return (
-            <ProjectInfoCardMinimal
-              style={{marginBottom: 20}}
-              key={item.projectId}
-              projectId={item.projectId}
-              onPress={handlePress}
-            />
-          );
-        }}
+        keyExtractor={p => p.projectId}
+        initialNumToRender={6}
+        windowSize={5}
+        removeClippedSubviews
+        renderItem={renderItem}
       />
     </View>
   );
 };
 
+export const ProjectCardLoader = () => (
+  <View
+    style={{
+      height: 112,
+      borderRadius: 12,
+      backgroundColor: '#eee',
+      marginBottom: 20,
+    }}
+  />
+);
+
```

Comment:
While i think this looks cool, I think we need approval from design before we can introduce this